resolvers/webpack/index.js

```diff
@@ -44,12 +49,16 @@ exports.resolve = function (source, file, settings) {
   }
 
   var webpackConfig
-
-  var configPath = get(settings, 'config')
-    , configIndex = get(settings, 'config-index')
+    , configPath
+    , configIndex
     , packageDir
 
-  log('Config path from settings:', configPath)
+  // Only respect webpackConfig settings if not in a node_module
```

Comment:
i'm not sure why this is something we'd want to do. webpack config applies to *every single JS file*, potentially, including everything inside a node_modules directory.

Reply:
That’s true, for rules, etc., but I wonder if it applies to aliases or other resolve configs, too? I would be surpsied since I can’t think why anyone would want to affect the module resolution within node_modules... and if webpack did do that then surely there would be tonnes of problems as a result..?

I’d also argue that since this is an eslint plugin, the user is never interested in node_modules, thus this change does not have an adverse affect?

Reply:
It applies to everything. Webpack is used *all the time* to modify the resolution of things within node_modules - Airbnb does it at times.

This eslint plugin checks across files - so it's interested in every JS file that's required/imported, full stop.

Reply:
Okay, so is the only solution to resolve the path relative to the user’s project? I can’t see any obvious way to do that..